app/src/main/java/com/ayush/weatherapp/home/HomeActivity.java

```diff
@@ -166,14 +168,12 @@
     return this;
   }
 
-  @Override public void showProgressDialog(String message, boolean cancelable) {
-    getProgressDialog().setMessage(message);
-    getProgressDialog().setCancelable(cancelable);
-    getProgressDialog().show();
+  @Override public void showSwipeRefresh() {
```

Comment:
showSwipeRefresh(boolean show) could be used to reduce the two functions into one.

Reply:
:ok: